frontend/src/components/HospitalInventoryEdit.tsx

```diff
@@ -0,0 +1,194 @@
+import React, { useEffect, useState } from 'react'
+import Col from 'react-bootstrap/Col'
+import Form from 'react-bootstrap/Form'
+import Card from 'react-bootstrap/Card'
+import Button from 'react-bootstrap/Button'
+import { useParams } from 'react-router-dom'
+import style from './HospitalInventoryEdit.module.css'
+import { HospitalInventory, HospitalAddress, fetchHospitalInventory, fetchHospitalAddress } from '../apis/infomed'
+
+
+interface HospitalInventoryParams {
+  hospitalId: string
+}
+
+export const HospitalInventoryEdit = ({hospitalId}: HospitalInventoryParams) => {
+
+  const [inventoryList, setInventoryList] = useState<HospitalInventory[]>([]);
+  const [hospitalAddress, setHospitalAddress] = useState<HospitalAddress[]>([]);
+
+  useEffect(() => {
+    (async () => {
+      const list = await fetchHospitalInventory(hospitalId);
+      setInventoryList(list);
+      
+      const address = await fetchHospitalAddress(hospitalId);
+      setHospitalAddress(address);
+    })();
+  }, [hospitalId]);
+
+  const handleInventoryChange  = (e: React.ChangeEvent<HTMLInputElement>): void => {
+    const updateList = [...inventoryList];
+    const target = e.target.getAttribute("data-index");
+    const dataIndex = parseInt(target !== null ? target : '0');
```

Comment:
There's a much easier way. Make the `index` a parameter to `handleInventoryChange()` and attach it as a handler this way:

```tsx
<Form.Control onChange={e => handleInventoryChange(index, e)}>
```